### PR TITLE
change title & code review for advance assignment of php

### PR DESCRIPTION
**Pull request by SouravPaul1999**

### CODE REVIEW

File: Advance_assignment.php [hunk 1]
```diff
@@ -35,44 +35,49 @@
...
-      echo "<div class=' col-sm-12 col-md-6  col-xl-4'>";
-
+      
+      echo "<div class=' col-sm-12 col-md-6  col-xl-6'>";
+      
       echo "<div class='text-break'>$stitle</div>";
-
-
+      
+
```
Comment (kunal-sin) [inline Advance_assignment.php]: Don't mix PHP and HTML.

Write HTML outside of php tag.

File: advance_mailer.php [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+      max-width: 330px;
+      padding: 15px;
+      margin: 0 auto;
+      color: #017572;
+      font-size: large;
+    }
+
+    .form-signin .form-control {
+      position: relative;
+      height: auto;
```
Comment (kunal-sin) [inline advance_mailer.php]: Create a separate CSS file for this.

Comment (SouravPaul1999) [reply advance_mailer.php]: issue fixed

Comment (SouravPaul1999) [reply advance_mailer.php]: > Create a separate CSS file for this.

fixed

File: Advance_assignment.php (hunk 1 above)
Comment (SouravPaul1999) [reply Advance_assignment.php]: > Don't mix PHP and HTML.
> 
> Write HTML outside of php tag.

done
@kunal-sin

**Review by kunal-sin — CHANGES_REQUESTED** [review]
Please do cleanup your code. Remove unnecessary / Not required codes.

**Pull request merged**